### PR TITLE
fix: links without protocol

### PR DESCRIPTION
**Pull request by zhuangya**

### CODE REVIEW

File: _signatures/Hashin_Jithu.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
   name: Hashin Jithu
-  link: 
+  link:
```
Comment (emhoracek) [inline _signatures/Hashin_Jithu.md]: Blank fields shouldn't listed, it should just be: 
```
---
  name: Hashin Jithu
---
```

File: _signatures/Max_Hudson.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
   name: Max Hudson
-  link: 
+  link:
```
Comment (emhoracek) [inline _signatures/Max_Hudson.md]: Same as  `_signatures/Hashin_Jithu.md `

**Comment by emhoracek** [comment]
Thanks for doing this!

**Comment by zestyping** [comment]
Thank you!

**Pull request merged**